spec/mongo/collection_spec.rb

```diff
@@ -1156,6 +1228,44 @@
         }.to raise_exception(Mongo::Error::OperationFailure)
       end
     end
+
+    context 'when bypassing document validation', if: find_command_enabled? do
+
+      before do
+        collection_with_validator.insert_one({:a => 1})
+      end
+
+      after do
+        collection_with_validator.delete_many
+      end
+
+      let(:result) do
+        collection_with_validator.replace_one({:a => 1}, {:a => 5})
+      end
+
+      it 'replaces successfully when document is valid' do
+        expect(result.modified_count).to eq(1)
+      end
+
+      let(:result2) do
+        collection_with_validator.replace_one({:a => 1}, {:x => 5})
+      end
+
+      it 'raises OperationFailure when replacement is invalid' do
+        expect {
+          result2
+        }.to raise_exception(Mongo::Error::OperationFailure)
+      end
+
+      let(:result3) do
+        collection_with_validator.replace_one(
+            {:a => 1}, {:x => 1}, :bypass_document_validation => true)
+      end
+
+      it 'replaces successfully when document is invalid but bypass_document_validation is true' do
```

Comment:
you don't have to say "but bypass_document_validation is true" because it's stated in the outer context. When the test runs, it'll read "when bypassing document validation it replaces successful when document is invalid"